src/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link href="vendors/bootstrap/css/bootstrap.min.css" rel="stylesheet">
```

Comment:
As I can see we are using only the bootstrap's CSS, I recommend taking a look at React-Bootstrap when we need the bootstrap framework. https://react-bootstrap.github.io/